src/Components/Browser.JS/package.json

```diff
@@ -5,14 +5,15 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "prebuild": "yarn install --mutex network",
```

Comment:
Curious - why was this added? While I think it's normal for someone who is used to `dotnet build` to expect `yarn run build` to also restore, I don't know if this is something that's normally done in NodeJS projects. Yarn is pretty good about no-op and caching, but let's be careful about unintended consequences for running restore during build.

Reply:
Think I could remove it now. When I was messing around with getting the linked build, I experimented with differing the `yarn install` until after project discovery. I'll try and undo.